understanding/22/focus-not-obscured-minimum.html

```diff
@@ -56,12 +56,21 @@ <h4>Modal dialogs</h4>
+
+         <ul>
+            <li><strong>When the additional content appears, it displaces existing content.</strong> An accordion is an example of this. When an accordion is opened, the disclosed content shifts existing content further down the page. Since the new content does not obscure existing content, it cannot obscure the item with focus.</li>
+            <li><strong>When the additional content appears, existing content reflows.</strong> The popout sidebar on the <a href="https://www.w3.org/TR/WCAG22/">WCAG standard</a> is an example of this pattern. When the side menu is activated, it discloses a new section of information along the left side of the page. The main content area is reduced horizontally to accommodate this, and the existing main content reflows to fit in the thinner space. As a result, there is no overlapping content between the two sections; the item receiving focus, whether in the left navigation or in the main content, will not be obscured by the other section.</li>
+            <li><strong>When the additional content is disclosed, it takes focus and the tab ring is constrained to the new content until it is dismissed.</strong> This modality is somewhat like a dialog, in that a user cannot navigate beyond the disclosed content by keyboard without dismissing it first (typically by pressing Esc). However, unlike in a modal dialog, in some implementations a pointer user may be able to interact with content outside the disclosed section without dismissing it. Since this pattern potentially creates an inequitable experience between keyboard and pointer users, it should be used cautiously. That said, it does prevent the disclosed content from obscuring the keyboard focus in the main content, and thus should pass this SC. </li>
+            <li><strong>The disclosure expands into an area of the page containing no other content.</strong> Many pages are designed with wide margins, providing significant white space into which new content can be disclosed. Many chatbots and toasters are designed to 'slide up' into the right unpopulated side of a page. Where authors are careful to ensure content is not obscured at each breakpoint in a response design, no obscuring of other operable content need occur.</li>
+            <li><strong>When focus leaves the additional content, it is hidden or collapsed.</strong> This is very similar to patterns discussed next under Non-persistent disclosed information. A distinguishing factor can be that the user's last point of interaction in the disclosure is preserved (it persists) even though it may be hidden until a user returns. Some trees and left navigation pattterns behave this way. </li>
+         </ul>
+         <p>Where disclosed content persists and causes existing content to be obscured, it will be at risk of failing this criterion if it prevents users from seeing the focus indicator.</p>
        </section>
        <section>
          <h4>Non-persistent disclosed information</h4>
```

Comment:
in the intro to this doc, non-persistent disclosures aren't mentioned, but get quite a bit written about them.  maybe worth alluding to?
>Typical types of content that can overlap focused items are sticky footers, sticky headers, and non-modal dialogs. As a user tabs through the page, these layers of content can obscure the item receiving focus, along with its focus indicator.